teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -637,7 +641,56 @@ The Artifact Hub provides a [generic solution](https://artifacthub.io/docs/topic
 
 ### Verified Remote Resources
 
-TODO
+#### Sign the Verified Catalog
+
+Following the design in TEP-0091, all the releases of Verified Catalog resources should be [signed](https://github.com/tektoncd/community/blob/main/teps/0091-trusted-resources.md#sign-the-resources) by the signing [tkn tool](https://github.com/tektoncd/cli) and the signature should be stored in the `tekton.dev/signature` annotation:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/signature: MEQCIHhFC480mv7mh/6dxAlp/mvuvGXanuSghMsT+iBhWxt5AiBDvHv8sfKjJ3Ozrzvp+cjr30AOx1SPQDCcaJjlpAoVrA==
+  name: example-task
+spec:
+  steps:
+  - args:
+    - Hello World!
+    command:
+    - echo
+    image: ubuntu
+    name: echo
+    resources: {}
+```
+
+The `tekton.dev/signature` annotation will be parsed by [Artifact Hub][hub] for every published Tekton resource. A `signed` badge will be displayed in the UI for every release of resource that contains the signature annotation, demonstrating that the integrity of the resource can be verified.
+
+![Artifact Hub Signed Badge](images/0079-signed-badge.png)
+
+The [verification](https://github.com/tektoncd/community/blob/main/teps/0091-trusted-resources.md#verify-the-resources) of the signature can be done in the Tekton Pipeline's reconciler during the remote resolution.
+
+A new signing key pair - `verified-catalog` will be generated using KMS to sign the Verified Catalogs, which will be hosted in the [`tekton-releases`](http://console.cloud.google.com/home/dashboard?project=tekton-releases) GCP project. The signing permission is only granted to the Tekton Catalog Maintainers.
```

Comment:
This project is not visible to public right? maybe add one comment that it is only visible to maintainers. 
We may also be clear that, only  `Cloud KMS Admin`
`Cloud KMS CryptoKey Signer/Verifier` and `Viewer` permissions should be given to maintainers. Since this is not a dedicated project.

Reply:
SG, just added the details. This can also be treated as implementation details.